SPECS/kernel/CVE-2023-46862.nopatch

```diff
@@ -0,0 +1,4 @@
+CVE-2023-46862 - Introducing commit not present
+Introducing patch not present. Note that 5.15 doesn't even have the vulnerable file https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/io_uring/fdinfo.c?h=v5.15.137
+upstream fix: 7644b1a1c9a7ae8ab99175989bfc8676055edb46
+But io_uring/fdinfo.c isn't introduced until a4ad4f748ea96202451f88b6385d7283d0f2e513; not backported to 5.15.*
```

Comment:
Judging by this [commit](https://github.com/torvalds/linux/commit/a4ad4f748ea96202451f88b6385d7283d0f2e513), the function was moved to a new file. However, we still have the vulnerable function `io_uring_show_fdinfo` in our [code](https://github.com/microsoft/CBL-Mariner-Linux-Kernel/blob/rolling-lts/mariner-2/5.15.137.1/io_uring/io_uring.c#L10414).

Reply:
Thanks for pointing this out. Shall we wait for LSG to backport the commit, or do we want to do that ourselves?

Reply:
Because it is Med let's wait for the backport in LTS